### ISSUE TITLE
[Accessibility] 1xBroken ARIA reference, 2xEmpty button and 1xMissing form label

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#5]
**Steps to reproduce:**

https://github.com/user-attachments/assets/b8e92f87-4b21-4e06-8e6a-25c45df54e66

**Actual Result:**

![Image](https://github.com/user-attachments/assets/7fab2ba4-9ff9-4612-9230-c8bf44c64702)

**Expected Result:**
No accessibility issues are present
